src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+
+* {
+  margin: 0;
+}
+
+.moyo-container {
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.moyo-logo1 {
+  padding: 10px 0;
+}
+
+.main-header {
+  max-width: 1200px;
+  display: flex;
+  justify-content: space-between;
+  border: none;
+  padding: 0 50px;
+}
+
+.main-header .main-nav-list {
+  display: flex;
+  justify-content: space-around;
+  padding: 19.5px 0;
+}
+
+.main-nav-list > li {
```

Comment:
don't use a combined selector when you can just add a class to the `li` tag and add styles using selector by class